instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -63,8 +63,16 @@ def add(x, y):
 from timeit import default_timer
 from typing import Collection, Iterable
 
+from billiard.einfo import ExceptionInfo
 from celery import signals  # pylint: disable=no-name-in-module
 
+try:
```

Comment:
This is a bit convoluted. If I understand correctly, this is done to handle different versions of `billiard`, right? (I mean some version of `billiard` have `billiard.einfo.ExceptionWithTraceback` and others don't, right?)

If that is the case, please check the version of `billiard` and use an `if`/`else` instead of `try`/`except`. For the reader is more clear this way what is being intended here.

Reply:
Done, though I'm not certain if it's much clearer. Moved the `if/else` to `utils.py`, so at least it's out of the way.